clients/src/main/java/org/apache/kafka/common/compress/KafkaLZ4BlockOutputStream.java

```diff
@@ -41,6 +41,10 @@ public final class KafkaLZ4BlockOutputStream extends OutputStream {
 
     public static final String CLOSED_STREAM = "The stream is already closed";
 
+    public static final int MIN_COMPRESSION_LEVEL = 1;
```

Comment:
Should we rename this class to `LZ4OutputStream` to be consistent with `GzipOutputStream`?

Reply:
The kafka naming convention would be `Lz4OutputStream`.